### PR TITLE
added --cdn to deploy task

### PR DESCRIPTION
**Pull request by andrewjtait**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -44,6 +44,10 @@ The following component have had minor internal changes to satisfy the introduct
 * TableHeader
 * TableRow
 
+## Deployment Changes
+
+You can now pass `--cdn` to the gulp task to bundle assets pointing towards the CDN.
+
```
Comment (ianoxley) [inline CHANGELOG.md]: Is this in the right place? (hat tip: @SeanArmstrong )

Comment (andrewjtait) [inline CHANGELOG.md]: fixed

**Comment by ianoxley** [comment]
When I run this locally I still get 4 images loaded from localhost:

localhost_images

**Comment by andrewjtait** [comment]
@ianoxley fixed

**Pull request merged**